staging/src/k8s.io/apiserver/pkg/endpoints/handlers/rest.go

```diff
@@ -465,8 +476,8 @@ type namedCreaterAdapter struct {
 	rest.Creater
 }
 
-func (c *namedCreaterAdapter) Create(ctx request.Context, name string, obj runtime.Object) (runtime.Object, error) {
-	return c.Creater.Create(ctx, obj)
+func (c *namedCreaterAdapter) Create(ctx request.Context, name string, obj runtime.Object, includeUninitialized bool) (runtime.Object, error) {
```

Comment:
However, if it's the same parameter everywhere in the system, including to users and code... it makes figuring out what the value means easier.  :)